Weather_vs_demand.ipynb

```diff
@@ -0,0 +1,541 @@
+{
```

Comment:
<p>We should be consistent in how we calculate the rolling mean for each date. When I did this before, I made sure the results were centered (i.e., the date assigned to the 7 day average was the middle of the date range; in this case, it's the end of the date range). Either one is valid, we just need to pick one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/emapple/social/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>